langchain/agents/character_chat/base.py

```diff
@@ -0,0 +1,133 @@
+"""An agent designed to hold a conversation with a character in addition to using tools."""
+from __future__ import annotations
+
+from typing import Any, List, Optional, Sequence, Tuple
+
+from pydantic import Field
+
+from langchain.agents.agent import Agent, AgentOutputParser
+from langchain.agents.character_chat.output_parser import ConvoOutputParser
+from langchain.agents.character_chat.prompt import (CHARACTER_SUMMARY, PREFIX,
+                                                    SUFFIX,
+                                                    TEMPLATE_TOOL_RESPONSE)
+from langchain.agents.utils import validate_tools_single_input
+from langchain.base_language import BaseLanguageModel
+from langchain.callbacks.base import BaseCallbackManager
+from langchain.chains import LLMChain
+from langchain.prompts.base import BasePromptTemplate
+from langchain.prompts.chat import (ChatPromptTemplate,
+                                    HumanMessagePromptTemplate,
+                                    MessagesPlaceholder,
+                                    SystemMessagePromptTemplate)
+from langchain.schema import (AgentAction, AIMessage, BaseMessage,
+                              BaseOutputParser, SystemMessage)
+from langchain.tools.base import BaseTool
+
+
+class CharacterChatAgent(Agent):
+    """An agent designed to hold a conversation in addition to using tools."""
```

Comment:
Would it be more accurate to rephrase this as "An agent designed to hold a conversation as a given character in addition to using tools."

Reply:
Same comment as above.  Incorporating.